release_notes/45-release-notes.adoc

```diff
@@ -0,0 +1,421 @@
+
+[id="build-time-network-policy-tools-updates_{context}"]
+=== Build-time network policy tools updates
+
+This release introduces an update to the `roxctl netpol generate` command.
+
+* By default, the command uses port `53` for DNS connections.
+* You can use the `--dnsport` option to override the default DNS port. If you are using {ocp}, you must always change the port when generating network policies with `roxctl` because {ocp} uses port `5353` by default. For example, `roxctl netpol generate --dnsport 5353 <other-options>`.
+
+To simplify usage for {ocp} and other systems that use named ports, Red{nbsp}Hat plans to enhance the `--dnsport` option in a future release to accept strings and numbers. With that change, the generated network policy can use a named port, such as `dns`, instead of a specific port number, which is a more portable approach.
+
+For more details, see xref:../cli/generating-build-time-network-policies.adoc#using-the-build-time-network-policy-generator_generating-build-time-network-policies[Generating build-time network policies].
+
+[id="enhanced-acscs-experience_{context}"]
+=== Enhanced {product-title-managed-short} experience
+
+We have improved the {product-title-managed-short} experience on `console.redhat.com`, making signing up for a trial and getting support easier.
+
+[id="notable-technical-changes_{context}"]
+== Notable technical changes
+
+* For {product-title-short} Operator, Red{nbsp}Hat has added a label selector for caching configuration of secrets and configmaps, resulting in a significant reduction in memory consumption, especially on large clusters. In testing, we have seen a 28% decrease in memory usage on a new {ocp} cluster.
+
+* The {product-title-short} Operator now adds a label `app.stackrox.io/managed-by: operator` to all Helm chart resources and secrets created by the operator, providing better organization and visibility.
+
+* The {product-title-short} Operator increases the number of requests sent to the API server to retrieve secrets not managed by the Operator and do not match the cache label selector.
+
+* Scanner DB now runs on PostgreSQL 15, replacing the earlier version, PostgreSQL 12. Since the database is not persisted, no migration is required, and you can continue using the scanner without any additional steps.
+
+* The Nexus and Red{nbsp}Hat registry integrations now attempt to pull manifest digests using a HEAD request to `/v2/<name>/manifests/<reference>`. This change resolves an issue that previously resulted in an `unsupported digest algorithm error` when using Scanner V4. You can turn off this new behavior by setting the environment variable `ROX_ATTEMPT_MANIFEST_DIGEST` to `false`.
+
+* {product-title-short} includes new policy categories, and some default policies are tagged with these new policy categories.
++
+.New policy categories
+[options="header"]
+|====
+
+|Policy category|Policy
+
+.2+|*Zero Trust*
+|Deployments should have at least one ingress Network Policy
```

Comment:
```suggestion
|Deployments need at least one ingress Network Policy
```

Reply:
These names are what we have in product currently so we cannot change these policy names. Ditto for other entries in this table